cmd/create/cluster/cmd.go

```diff
@@ -1403,6 +1443,28 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(0)
 	}
 
+	if isSTS {
+		if mode != "" {
+			operatorroles.Cmd.Run(operatorroles.Cmd, []string{
+				clusterName,
+				mode,
+				permissionsBoundary})
+			oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode})
+		} else {
+			rolesCMD := fmt.Sprintf("rosa create operator-roles --cluster %s", clusterName)
+			oidcCMD := fmt.Sprintf("rosa create oidc-provider --cluster %s", clusterName)
+
+			if permissionsBoundary != "" {
+				rolesCMD = fmt.Sprintf("%s --permissions-boundary %s", rolesCMD, permissionsBoundary)
+			}
+
+			reporter.Infof("Run the following commands to continue the cluster creation:\n\n"+
+				"\t%s\n"+
+				"\t%s\n",
+				rolesCMD, oidcCMD)
```

Comment:
This entire block should be at the very end, after calling `describe cluster` - otherwise it gets lost in the scroll